doc/docs/master/how-tos/basic-data-operations/sql-ingest.md

```diff
@@ -0,0 +1,133 @@
+
+# Ingest Structured Data - Integration with SQL
+
+Part of your data might live in databases requiring some level of integration with your warehouse to retrieve and inject them into Pachyderm.
+
+Our **SQL ingest** tool provides a seamless connection between databases and Pachyderm,  allowing you to import data from a SQL database into Pachyderm-powered pipelines. By bringing data-driven pipelines, versioning & lineage to structured data, we are allowing Data Science teams to easily combine structured and unstructured data, thus improving the performance of prediction models.
+
+Specifically, we help you connect to a remote database of your choice and pull the result of a given query at regular intervals in the form of a CSV or a JSON file.  
+
+## Use SQL Ingest
+Pachyderm's SQL Ingest uses [pipeline templates](../../pipeline-operations/pipeline-templates) populated with the following parameters to automatically create the pipelines that access, query, and materialize the results in the form of CSV or JSON files.
+
+Pass in the following parameters and get your results committed to an output repo, ready for the following downstream pipeline:
+```shell
+pachctl update pipeline --jsonnet ./src/templates/sql_ingest_cron.jsonnet
```

Comment:
This should be the url to master when we merge.  Maybe we can change it to the release commit url at some point

Reply:
Set `https://raw.githubusercontent.com/pachyderm/pachyderm/{{ config.search_index_version }}/src/templates/sql_ingest_cron.jsonnet ` instead where {{ config.search_index_version }} comes from the config file.
i.e: `master` now, `2.1.x` once released.